test/util/nightwatch-commands/selectDropdown.js

```diff
@@ -20,7 +20,7 @@ exports.command = function selectDropdown(name, value) {
     element.value = clientValue;
     element.dispatchEvent(evt);
     return element.value;
-    /* eslint-disable */
+    /* eslint-enable */
```

Comment:
Has nothing to do with this PR, but I noticed it probably wasn't supposed to be disabled twice. :man_shrugging: